### PR TITLE
Added 2021/Dundee/OMERO.figure-workshop

### PR DESCRIPTION
**Pull request by will-moore**
OME 2021 OMERO.figure workshop slides

### CODE REVIEW

**Comment by joshmoore** [comment]
Is this currently deployed anywhere?

**Comment by will-moore** [comment]
Just pushed to gh-pages at http://will-moore.github.io/presentations/2021/Dundee/OMERO.figure-workshop/

File: 2021/Dundee/OMERO.figure-workshop/index.html [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+                    <ul>
+                        <li>Main content is in <a target="_blank" href="https://omero-guides.readthedocs.io/en/latest/">OMERO guides</a>
+                        </li>
+                        <img src="images/guide1.png" />
+                        <img src="images/guide3.png" class="guides1" />
+                        <img src="images/guide4.png" class="guides2" />
+                        <img src="images/guide5.png" class="guides3" />
+                        <a target="_blank" href="https://omero-guides.readthedocs.io/en/latest/">https://omero-guides.readthedocs.io</a>
+                        <li>Watch videos on <a target="_blank"
+                                href="https://www.youtube.com/channel/UCyySB9ZzNi8aBGYqcxSrauQ/videos/">openmicroscopy YouTube
```
Comment (joshmoore) [inline 2021/Dundee/OMERO.figure-workshop/index.html]: https://www.youtube.com/c/OpenMicroscopyEnvironment works as well

File: 2021/Dundee/OMERO.figure-workshop/index.html [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+                        <li><i>Create</i> figures with Python scripts on the server</li>
+                        <li><i>Edit</i> figures with JavaScript in the client (web browser)</li>
+                        <img style="width: 550px; margin-left:100px" src="images/omero-figure-scripts.png" />
+                    </ul>
+                </section>
+
+                <section data-state="ome-theme" data-background="images/figure-scripting-idr0079.png">
+                    <a href="https://github.com/ome/omero-figure/issues/431" target="_blank"
+                        style="position: absolute; right: 0; bottom: -600px;"
+                    >More Info</a>
```
Comment (joshmoore) [inline 2021/Dundee/OMERO.figure-workshop/index.html:173]: I like this!

File: 2021/Dundee/OMERO.figure-workshop/index.html [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+                    </div>
+                </section>
+
+                <section data-state="ome-theme" style="width:120%; left:-60%">
+                    <h2><a target="_blank" href="https://idr.openmicroscopy.org/">idr.openmicroscopy.org</a></h2>
+                    <img src="images/idr.png" width="1900px"/>
+                </section>
+
+                <section data-state="ome-theme" style="color: white" data-background="images/idr0079-pLLP_in_whole_embryo.png">
+                    <h2 style="color: white">Today's workshop:</h2>
```
Comment (joshmoore) [inline 2021/Dundee/OMERO.figure-workshop/index.html:130]: This page almost makes me think to generate a reveal.js for each study ;)

cc: @sbesson @francesw

**Review by joshmoore — COMMENTED** [review]
Nice!

**Comment by jburel** [comment]
eosc logo should be added to the presentation
The easiest is probably to push a new logo to directly to master since we need the logo in other presentation too

**Comment by jburel** [comment]
The logo is already there it is named ``eosc-life.png``

**Comment by will-moore** [comment]
I was going to add https://eosc-portal.eu/sites/all/themes/theme1/logo.png, but I can use the `eosc-life.png`.

**Comment by jburel** [comment]
The version currently there has a border, I will push a commit to  master to replace it
I will keep the same name

**Comment by will-moore** [comment]
Are there any logos we can remove? Otherwise it's going to need a bit more work on layout, since just adding one to the list doesn't fit any more.

**Comment by jburel** [comment]
I think Corbel and Multimot could be removed

**Comment by jburel** [comment]
I have replaced the eosc-life logo

**Comment by will-moore** [comment]
http://will-moore.github.io/presentations/2021/Dundee/OMERO.figure-workshop/ updated.
@joshmoore, @jburel Sorry, I didn't re-list this for today, but if you're happy with those last tweaks, please feel free to merge.
 Thanks.

**Comment by joshmoore** [comment]
:+1: from my side.

**Comment by jburel** [comment]
Thanks @will-moore

**Pull request merged**